src/encoded/static/components/genome_browser.js

```diff
@@ -831,6 +831,7 @@ class GenomeBrowser extends React.Component {
         const domain = `${window.location.protocol}//${window.location.hostname}`;
         files = this.sortFiles(primarySort, sortDirection, sortIdx, toggleFlag).filter((file) => file.assembly === this.props.assembly);
         newFiles = [...this.state.pinnedFiles, ...files];
+        newFiles = _.uniq(newFiles, (file) => file.href);
```

Comment:
It's late in the game and I recommend you don't change this. Also, this is my personal feeling. I feel it is best to let go of ES 5 techniques in favor of ES2015+ techniques. _.uniq does what the inbuilt set does and inbuit functionality is generally faster than third party tool.

Reply:
I can update this in ENCD-5618. Thank you!